src/api/app/views/webui/status_messages/new.html.haml

```diff
@@ -0,0 +1,28 @@
+- @pagetitle = 'Create Status Message'
+.row
+  .col
+    .card
+      .card-body
+        .row
+          .col-12
+            %h3= @pagetitle
+          .col-12
+            %p With "announcement" severity, the message will be displayed on the top and all over the web site and requires acknowledgement.
```

Comment:
This should be closer to the select. Like https://getbootstrap.com/docs/4.5/components/forms/#help-text

Reply:
I like the idea